$NetBSD: patch-configure,v 1.1 2012/06/05 15:27:22 wiz Exp $

Fix build breaker in NetBSD<6's sh:
checking if cc -std=gnu99 supports -errwarn=E_NO_EXPLICIT_TYPE_GIVEN -errwarn=E_NO_IMPLICIT_DECL_ALLOWED...
eval: 1: Syntax error: Bad substitution

--- configure.orig	2012-03-03 01:52:25.000000000 +0000
+++ configure
@@ -13733,7 +13733,7 @@ $as_echo "$supported" >&6; }
 
 		{ $as_echo "$as_me:${as_lineno-$LINENO}: checking if $CC supports -errwarn=E_NO_EXPLICIT_TYPE_GIVEN -errwarn=E_NO_IMPLICIT_DECL_ALLOWED" >&5
 $as_echo_n "checking if $CC supports -errwarn=E_NO_EXPLICIT_TYPE_GIVEN -errwarn=E_NO_IMPLICIT_DECL_ALLOWED... " >&6; }
-		cacheid=`$as_echo xorg_cv_cc_flag_-errwarn=E_NO_EXPLICIT_TYPE_GIVEN -errwarn=E_NO_IMPLICIT_DECL_ALLOWED`
+		cacheid=`$as_echo "xorg_cv_cc_flag_-errwarn=E_NO_EXPLICIT_TYPE_GIVEN -errwarn=E_NO_IMPLICIT_DECL_ALLOWED"`
 		if { as_var=`$as_echo "$cacheid" | $as_tr_sh`; eval \${$as_var+:} false; }; then :
   $as_echo_n "(cached) " >&6
 else
